infra/helper.py

```diff
@@ -148,6 +148,7 @@ def build_fuzzers(build_args):
 
   command = [
         'docker', 'run', '--rm', '-i',
+        '-e', 'SWITCH_UID=' + os.getuid(),
```

Comment:
Doesn't `os.getuid()` return an integer? If yes, let's do `'SWITCH_UID=%d' % os.getuid()` + let's use format string on line 154 instead of concatenation.